error: linking with `mips-openwrt-linux-musl-gcc` failed: exit status: 1
...
  = note: .../deps/dexios-606820e282cba8d2.dexios.e44c5fe829c7491a-cgu.0.rcgu.o: in function `getrandom::util_libc::open_readonly':
          dexios.e44c5fe829c7491a-cgu.0:(.text._ZN9getrandom9util_libc13open_readonly17h575927814a896b0aE+0x1c): undefined reference to `open64'
          collect2: error: ld returned 1 exit status

---

--- a/dexios/Cargo.toml
+++ b/dexios/Cargo.toml
@@ -29,3 +29,7 @@ anyhow = "1.0.62"
 
 zip = { version = "0.6.2", default-features = false, features = ["zstd"] }
 rpassword = "7.0"
+
+[dev-dependencies]
+# musl+getrandom
+libc = "=0.2.172"
